pkgs/development/python-modules/fuzzywuzzy/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, python, fetchFromGitHub, python-Levenshtein, pycodestyle, hypothesis, pytest }:
```

```suggestion
{ stdenv, buildPythonPackage, fetchFromGitHub, python-Levenshtein, pycodestyle, hypothesis, pytest }:
```

Don`t see `python` used anywhere in here.